File: event/src/test/java/de/danoeh/antennapod/event/settings/SkipIntroEndingChangedEventTest.java
package de.danoeh.antennapod.event.settings;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class SkipIntroEndingChangedEventTest {

    @Test
    public void skipIntroSkipEndingAndFeedIdAreStored() {
        SkipIntroEndingChangedEvent event = new SkipIntroEndingChangedEvent(5, 10, 7L);
        assertEquals(5, event.getSkipIntro());
        assertEquals(10, event.getSkipEnding());
        assertEquals(7L, event.getFeedId());
    }
}

File: event/src/test/java/de/danoeh/antennapod/event/settings/SkipIntroEndingChangedEventTest.kt
package de.danoeh.antennapod.event.settings

import org.junit.Assert.assertEquals
import org.junit.Test

class SkipIntroEndingChangedEventTest {

    @Test
    fun skipIntroSkipEndingAndFeedIdAreStored() {
        val event = SkipIntroEndingChangedEvent(5, 10, 7L)
        assertEquals(5, event.skipIntro)
        assertEquals(10, event.skipEnding)
        assertEquals(7L, event.feedId)
    }
}
